app/(pages)/admin/dashboard/layout.tsx

```diff
@@ -1,19 +1,36 @@
 "use client";
 import { client } from "@/app/contract-utils";
 import { MonetWorkLogo } from "@/components/icons/monet-work-logo";
+import { Button } from "@/components/ui/button";
+import { apiService } from "@/services/api.service";
+import { useMutation } from "@tanstack/react-query";
 import Link from "next/link";
 import { usePathname } from "next/navigation";
+import { toast } from "sonner";
 import { ConnectButton, useActiveAccount } from "thirdweb/react";
 import { createWallet } from "thirdweb/wallets";
 
 export default function Layout({ children }: { children: React.ReactNode }) {
   const activeAccount = useActiveAccount();
   const pathname = usePathname();
+  const SyncApiMutation = useMutation({
+    mutationFn: apiService.syncPoints,
+  });
+  const handleClick = () => {
+    SyncApiMutation.mutate(pathname as unknown as void, {
```

Comment:
please change name to syncApiMutation (camel cased with first character lower cased)